ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -80,6 +76,12 @@ public List<DataGridView> GetDGVs()
 
         public void ClearListsTablesFigures(bool clear_following)
         {
+            //clear all save acceptance actions --> will re-add save actions from loaded actions if peak still exists
+            Sweet.save_actions.RemoveAll(x => x.StartsWith("accept ExperimentalTheoretical") || x.StartsWith("unaccept ExperimentalTheoretical"));
+            //if in this round or others haven't ever shifted a mass, clear them all. Need to be careful because rerun the gamut whenever shift peaks.
+            if (!Sweet.lollipop.raw_experimental_components.Any(c => c.manual_mass_shift > 0))
```

Comment:
This checks whether peaks have already been shifted, correct? So remove any previous shifts, if there already were some? What if I did successive peak shifts? Shifted a peak, refresh, shifted another, refresh. Both of those shifts should be kept. I think they're erased here.